inst/tests/tests.Rraw

```diff
@@ -13775,9 +13775,9 @@ if (.Platform$OS.type=="windows") {
   test(1966.4, list.files(path = pth, pattern = "\\.csv$"), f)
   unlink(c(fp, file.path(pth, "\u00c3\u00b6.csv")))
   p = file.path(pth, "\u00fc"); dir.create(p); f = tempfile(tmpdir = p)
-  test(1966.5, fwrite(DT, enc2native(f)), NULL)
+  test(1966.5, filepath <- fwrite(DT, enc2native(f)), filepath)
```

Comment:
this case is more interesting, but note the test as you've written it is patently useless :)

In essence, you've written

```r
test(N, x, x)
```

That will always trivially be true!

Instead, we want `fwrite(DT, filepath<-enc2native(f)) to be compared to `filepath`.

Ditto the next test.